imports/plugins/included/marketplace/client/templates/settings/sellerShopSettings.js

```diff
@@ -83,47 +80,39 @@ Template.sellerShopSettings.helpers({
   },
 
   currencyOptions() {
-    const instance = Template.instance();
+    const sellerShop = Reaction.getSellerShop();
 
-    if (instance.subscriptionsReady()) {
-      const sellerShop = Reaction.getSellerShop();
+    if (!sellerShop) {
+      return;
+    }
 
-      if (!sellerShop) {
-        return;
-      }
+    const currencies = [];
+    for (const currency in sellerShop.currencies) {
+      if ({}.hasOwnProperty.call(sellerShop.currencies, currency)) {
```

Comment:
This pattern is clever and brief, but may be less understandable than using something like `Object.prototype.hasOwnProperty.call`